client/src/pages/RegisterPage.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
+import Input from '../components/Input';
+import Button from '../components/Button';
+import LogoImg from '../assets/icons/logo.svg';
+
+const LogoBox = styled.div`
+  display: flex;
+  align-items: baseline;
+  margin: 0 0 60px 0;
+`;
+
+const Logo = styled.div`
+  width: 44px;
+  height: 44px;
+  margin: 7px;
+  background-image: url(${LogoImg});
+`;
+
+const Name = styled.span`
+  margin: 7px;
+  font-family: 'Ornitons Heavy Regular';
+  font-size: 60px;
+  color: ${colors.textActiv};
+`;
+
+const Title = styled.span`
+  margin: 0 0 45px 0;
+  font-family: SF Pro Display Regular;
+  font-size: 23px;
+  color: ${colors.textInfoDark};
+`;
+const Card = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100vw;
+  height: 100vh;
+  background-color: ${colors.bgDark};
+`;
+
+const SignUpButton = styled(Button)`
+  margin-top: 52px;
+`;
+
+const ContainerLink = styled.div`
+  display: flex;
+  align-items: center;
+  font-family: SF Pro Display Regular;
+  font-size: 18px;
+`;
```

Comment:
This component is having the same styles as your component `Container` from `LoginPage.js`. To reduce the amount of code needed, you are able to move this component to a seperate file, give it a generic name like `Container` and use it on multiple pages if needed.